lib/utils.js

```diff
@@ -30,4 +32,16 @@ const getAgentInfo = () => ({
   name: pjson.name,
 });
 
-module.exports = { getCodeRef, getAgentInfo };
+const parseStringToArray = (str) => {
+  if (typeof str === typeof TYPE_STRING) {
+    return str.split(';').map((attribute) => {
+      const keyAndValue = attribute.split(':');
```

Comment:
Seems like it can be just `value`..
Please rename it to just `attribute`, and Iterator parameter to `attributeStr` or smth like that.